code/drasil-code/lib/Language/Drasil/Choices.hs

```diff
@@ -38,7 +38,9 @@ data Choices = Choices {
   -- | Constraint violation behaviour. Exception or Warning.
   srsConstraints :: Constraints,
   -- | List of external libraries what to utilize 
-  extLibs :: [ExtLib]
+  extLibs :: [ExtLib],
+  -- | Number of folders to go up in order to obtain the image
+  folderVal :: Int
```

Comment:
I think the original intent of the design was that there would be separate record type passed when generating code that would be responsible for specifying the environment instead of adding it to the `Choices` record here - I'm honestly not sure how important this distinction is at this point so I'm going to defer this to @JacquesCarette

Reply:
I was under the impression that I needed to submit a fix for the `ReadMe`s right now and that the creation of the record would be a separate much bigger issue which is why I created issue #3468. @JacquesCarette Do you think that I should look at creating a record for this PR itself or should the creation of the record be a separate issue?

Reply:
Oh I see - I didn't see that issue! Disregard this then lol